templates/upstart.conf.j2

```diff
@@ -7,4 +7,4 @@ respawn
 setuid {{ gunicorn_user }}
 chdir {{ gunicorn_app_dir }}
 
-exec gunicorn --config /etc/gunicorn/{{ gunicorn_app_name }}.py {{ gunicorn_wsgi }}
+exec gunicorn --config /etc/gunicorn/{{ gunicorn_app_name }}.py {{ gunicorn_wsgi }} {% if gunicorn_syslog %} 2>&1 | logger -t {{ gunicorn_app_name }} {% endif %}
```

Comment:
Using the example project to test, it doesn't look like anything is getting piped to `logger` here when `accesslog` and `errorlog` are left with their default values. If `syslog` is going to be left as the default, I think it might make sense to default the two log settings to `-`.